packages/storybook/stories/components/radio/technical-information.mdx

```diff
@@ -0,0 +1,61 @@
+import { Canvas, Meta, Markdown } from '@storybook/blocks';
+import SpecificationsRadio from '@ovhcloud/ods-components/src/components/radio/documentation/spec.md?raw';
+import { Banner } from '../../banner';
+import * as RadioStories from './radio.stories';
+
+<Meta of={ RadioStories } name='Technical information' />
+
+<Banner of={ RadioStories } />
+
+## Overview
+
+Radio are used when, a list of two or more options, the user must select only one option.
```

Comment:
Radio are used when, facing a list of two or more options, the user must select only one.